config/ext/rails.yml

```diff
@@ -330,6 +362,17 @@ Rails/Present:
   # Convert usages of `unless blank?` to `if present?`
   UnlessBlank: true
 
+Rails/RakeEnvironment:
+  Description: 'Include `:environment` as a dependency for all Rake tasks.'
+  Enabled: true
+  Safe: false
+  VersionAdded: '2.4'
+  Include:
+    - '**/Rakefile'
+    - '**/*.rake'
+  Exclude:
+    - 'lib/capistrano/tasks/**/*.rake'
+
```

Comment:
@foobear I'm not sure why `Rails/RakeEnvironment` should be useful. Aren't there rake task in a Rails project, that do not depend on Rails?

Reply:
I believe we do have the occasional Rake task which would not require the Rails environment, e.g. to generate an icon font. It only happens rarely, but I believe we should just disable this cop, don't see much benefit.